src/components/GlobalProvider/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { createContext, useMemo, useReducer } from 'react';
+import PropTypes from 'prop-types';
+import { globalReducer, initialState } from '../../data/reducers/global';
+
+export const GlobalContext = createContext();
+
+const GlobalContextProvider = ({
+  children,
+}) => {
+  const [globalState, dispatch] = useReducer(globalReducer, initialState);
+  const globalContext = useMemo(() => ({
+    ...globalState,
+    // Offsets by an additional 1 rem to avoid rendering the scrollbar unnecessarily
```

Comment:
Offset the viewheight by an additional 1rem to avoid the scrollbar rendering when the main content does not overflow.

Before 1rem offset:
https://github.com/openedx/frontend-app-admin-portal/assets/82611798/828bf096-1ae9-4df2-817b-21260ee40854

After 1rem offset:
https://github.com/openedx/frontend-app-admin-portal/assets/82611798/4d507e9d-3d19-4c8d-9aae-7f63d2bc954a